README.md

```diff
@@ -27,14 +28,14 @@
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/AngomRanjan/CapstoneCyberSecurity/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
+- Cindy Shin in Behance. For the Design Template
```

Comment:
Appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) in your Acknowledgments section, as required by the [Creative Commons License](https://creativecommons.org/licenses/) ✔